charts/incubator/teamspeak/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 3.13.6
 description: TeamSpeak Server
 name: teamspeak
-version: 0.3.0
+version: 0.4.1
```

```suggestion
version: 0.4.0
```

Bumping the chart to 0.4.0 should suffice